### DISCUSSION TITLE
Renovate doesn't seem to be creating the dependency dashboard nor updating specific app

### DISCUSSION THREAD

**Discussion by samip5** [renovatebot/renovate#22973] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.115.2

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I have two problems but i don't know if both should be in separate threads.

Problem 1: The dependency dashboard doesn't seem to be created after it had existed at one point in time, and as it wasn't getting re-opened by Renovate, I deleted it completely and now I don't have it at all. The log doesn't seem to suggest it trying to create it.
Problem 2: I don't seem to be getting Immich package updates after closing the PR, and I don't know what does it mean when Renovate lists things as "inactive branches" when those don't actually exist on the repository?

I don't believe that these are possible to replicate in reproduction, so I haven't created them.

### Relevant debug logs

<details><summary>Logs</summary>

Not sure about relevance so full debug logs are here: https://gist.github.com/samip5/b2c25cde26a239ecd159f6ad78e1f6ac

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Dependency dashboard is not updated after any run where an automerge happened

**Reply by rarkins** [reply]
Renovate also exits early after an automerge so maybe that's why the renovate/immich branch isn't getting reached

**Reply by rarkins** [reply]
I think it's best if you capture a run where an automerge _doesn't_ happen

**Comment by rarkins** [comment]
There's hundreds of hits for the string `Immich` so I can't really debug that from logs easily

**Comment by samip5** [comment, marked as answer]
I think my issue was mostly just that I needed to run Renovate really many times before the dashboard showed up.
